test/testenv/BUILD

```diff
@@ -0,0 +1,38 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test", "go_binary")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "testenv.go",
+    ],
+    deps = [
+        "//cmd/server/cmd:go_default_library",
+        "//cmd/shared:go_default_library",
+        "//pkg/handler:go_default_library",
+        "//template:go_default_library",
```

Comment:
Does this mean adapters can only test againt the built in templates ? It seems like the supported templates should be passed in from the test and not be part of the test environment

Reply:
We can make that possible.

Reply:
Done